clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/DeliverArraysExecutor.scala

```diff
@@ -22,24 +22,57 @@ class DeliverArraysExecutor(deliverCommand: DeliverArrays)(implicit ec: Executio
     extends DeliverExecutor(deliverCommand) {
 
   override protected[dispatch] def buildDelivery(
-    deliveredMetadata: ArraysMetadata,
-    moveOps: immutable.Seq[IoOp]
+    deliveredMetadata: ArraysMetadata
```

Comment:
I'd call this `existingMetadata` to be clear the method is transforming pre-delivery state into post-delivery state.

Reply:
check